internal/clients/manager.go

```diff
@@ -48,6 +49,8 @@ type AdminAPIClientsManager struct {
 	discoveredAdminAPIsNotifyChan    chan []adminapi.DiscoveredAdminAPI
 	gatewayClientsChangesSubscribers []chan struct{}
 
+	dbMode string
```

Comment:
I've created an issue to add a strongly typed enum for this (and other places we pass `dbMode` to): https://github.com/Kong/kubernetes-ingress-controller/issues/5013